### PR TITLE
Keycloak refactoring referred to Issue #5330 review

### PR DESCRIPTION
**Pull request by shekhar16**
#5330

### CODE REVIEW

**Review by yurem — APPROVED** [review]

**Review by yurem — APPROVED** [review]

**Review by yurem — APPROVED** [review]

File: jans-keycloak-link/model/pom.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>jans-keycloak-link-parent</artifactId>
+        <groupId>io.jans</groupId>
+        <version>1.0.17-SNAPSHOT</version>
```
Comment (yuriyz) [inline jans-keycloak-link/model/pom.xml]: Current version is 1.0.19-SNAPSHOT

File: jans-keycloak-link/pom.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.jans</groupId>
+  <artifactId>jans-keycloak-link-parent</artifactId>
+  <version>1.0.17-SNAPSHOT</version>
```
Comment (yuriyz) [inline jans-keycloak-link/pom.xml]: Current version is 1.0.19-SNAPSHOT

File: jans-keycloak-link/server/pom.xml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>jans-keycloak-link-parent</artifactId>
+        <groupId>io.jans</groupId>
+        <version>1.0.17-SNAPSHOT</version>
```
Comment (yuriyz) [inline jans-keycloak-link/server/pom.xml]: Current version is 1.0.19-SNAPSHOT

File: jans-keycloak-link/server/weld-validation-report.html [hunk 4]
```diff
@@ -0,0 +1,307 @@
+<html><head><title>Weld - Validation Report</title><style>body {
```
Comment (yuriyz) [inline jans-keycloak-link/server/weld-validation-report.html]: What is the purpose of this file in the project ?

File: jans-keycloak-link/service/pom.xml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jans-keycloak-link-parent</artifactId>
+        <groupId>io.jans</groupId>
+        <version>1.0.17-SNAPSHOT</version>
```
Comment (yuriyz) [inline jans-keycloak-link/service/pom.xml]: Current version is 1.0.19-SNAPSHOT

**Review by yuriyz — CHANGES_REQUESTED** [review]
Version has to be in sync with main otherwise it will be out of sync with other projects in mono repo. Current version is  1.0.19-SNAPSHOT.

File: jans-keycloak-link/service/pom.xml (hunk 5 above)
Comment (shekhar16) [reply jans-keycloak-link/service/pom.xml]: All changes are done.

**Review by yuriyz — DISMISSED** [review]

**Review by yurem — APPROVED** [review]

**Review by yurem — APPROVED** [review]

**Review by yuriyz — APPROVED** [review]

**Pull request merged**